blog/_posts/2021-07-19-scala-3.0.2RC1-is-here.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: blog-detail
+post-type: blog
+by: Michał Pałka, VirtusLab
+title: Scala 3.0.1 and 3.0.2-RC1 are here!
+---
+
+Greetings from the Scala 3 team! We are glad to announce that Scala 3.0.1 and 3.0.2-RC1 are now officially out.
```

Comment:
Would we be able to add here a reminder of the release schedule for the next final release and next RC? there have been a few requests for this in the contributors' forum

Reply:
What timeline should we set here then? 6 weeks from now? 6 weeks from the date of publishing artifacts? Basically this release took slightly longer than usually?

Reply:
6 weeks from the published artefacts sounds good. If the release doesn't have significant delays, we can disregard them, but since this one was more than a week, we need to offset.